hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -493,14 +493,8 @@ public void onCompactionCompleted(RocksDB db,
         builder = new CompactionLogEntry.Builder(trxId,
             System.currentTimeMillis(),
             inputFileCompactions.keySet().stream()
-                .map(inputFile -> {
-                  if (!inflightCompactions.containsKey(inputFile)) {
-                    LOG.warn("Input file not found in inflightCompactionsMap : {} which should have been added on " +
-                            "compactionBeginListener.",
-                        inputFile);
-                  }
-                  return inflightCompactions.getOrDefault(inputFile, inputFileCompactions.get(inputFile));
-                })
+                .map(inputFile -> inputFileCompactions.containsKey(inputFile) ?
+                    inflightCompactions.get(inputFile) : inputFileCompactions.get(inputFile))
```

Comment:
@swamirishi , The logic is incorrect -- if inputFileCompactions contains inputFile, shouldn't get it from inputFileCompactions?

Also, why double accessing the map (containsKey and then get)?

```java
                .map(inputFile -> {
                  final CompactionFileInfo f = inflightCompactions.get(inputFile);
                  return f != null ? f : inputFileCompactions.get(inputFile);
                })
```